modules/otel-troubleshoot-metrics.adoc

```diff
@@ -19,26 +19,59 @@ The Operator creates a `<cr_name>-collector-monitoring` telemetry service that y
 
 .Procedure
 
-. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource:
+. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource (CR):
 
 +
 [source,yaml]
 ----
+# ...
   config: |
     service:
       telemetry:
         metrics:
           address: ":8888" # <1>
+# ...
 ----
 <1> The address at which the internal collector metrics are exposed. Defaults to `:8888`.
 
-// TODO Operator 0.82.0 has spec.observability.metrics.enableMetrics config that creates ServiceMonitors for users
-
 . Retrieve the metrics by running the following command, which uses the port-forwarding Collector pod:
 +
 [source,terminal]
 ----
 $ oc port-forward <collector_pod>
 ----
 
-. Access the metrics endpoint at `+http://localhost:8888/metrics+`.
+. In the `OpenTelemetryCollector` CR, set the `enableMetrics` field to `true` to scrape internal metrics by using `PodMonitors` or `ServiceMonitors` depending on the deployment mode of the OpenTelemetry Collector:
+:
++
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+spec:
+# ...
+  mode: deployment
+  observability:
+    metrics:
+      enableMetrics: true # <1>
+# ...
+----
++
+[NOTE]
+====
+Alternatively, if you don't set the `enableMetrics` field to `true`, you can access the metrics endpoint at `+http://localhost:8888/metrics+`.
+====
+
+. In the *Observe* page in the web console, enable *User Workload Monitoring* to visualize the scraped metrics.
```

Comment:
Just checking - is there any prerequisite that needs to be added to this procedure since it's now instructing users to perform actions on the web console?

Reply:
The web console is used for viewing the metrics. It's just making use of an existing functionality. The whole instance is installable by using the web console, and a number of this product's features use the web console, so it is common for tasks to be performed by using the web console for this product.